City-of-Helsinki/helfi-etusivu-news-search
# Fix app install

Fixed dependencies + index.tsx to allow the app to compile properly.

To test:
- `nvm use; npm i; npm start`. The app should compile.
-  Run `npm t`. Run all tests. Tests should pass.
